2021q1/linuxulator.md

```diff
@@ -0,0 +1,30 @@
+## Linux compatibility layer update ##
+
+Contact: Edward Tomasz Napierala, <trasz@FreeBSD.org>
+
+Linuxulator improvements have been ongoing for the last two years,
+with support from the FreeBSD foundation over a few distinct project
+grants as well as contributions from the community.
+The goal of this project is to improve FreeBSD's ability to execute
+unmodified Linux binaries.
+Current status is being tracked at [Linux app status Wiki page](https://wiki.freebsd.org/LinuxApps).
+
+The work this quarter focused on making sure the 13.0-RELEASE ships
+with Linuxulator in a good shape, and fixing problems reported by
+users.  There are some new directories provided by `linsysfs(5)`,
+the lack of which, through a curious chain of events, broke
+installation of make(1) in Ubuntu Focal.  The `getcwd(2)` syscall
+was fixed to no longer return the wrong error value, which was
```

Comment:
we might want to indicate under what condition the wrong error value was returned? Or just "wrong error value for certain conditions" or so

Reply:
It returned ENOMEM instead of ERANGE when the buffer size passed to getcwd(2) was too small, which can mean either long paths, or just application scaling the buffer in some weird way.  I think I'll go with "certain conditions".